s/sentry-python/sentry-python_ubi_9_3.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash -e
+# ----------------------------------------------------------------------------
+#
+# Package       : sentry-python
+# Version       : 2.7.1
+# Source repo   : https://github.com/getsentry/sentry-python
+# Tested on     : UBI: 9.3
+# Language      : Python
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Stuti Wali <Stuti.Wali@ibm.com>
+#
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+set -e
+
+#variables
+PACKAGE_NAME=sentry-python
+PACKAGE_VERSION=${1:-2.7.1}
+PACKAGE_URL=https://github.com/getsentry/sentry-python
+
+sudo dnf install -y ncurses wget git python3 python3-devel make gcc gcc-c++
+
+# miniconda installation
+wget https://repo.anaconda.com/miniconda/Miniconda3-py311_23.10.0-1-Linux-ppc64le.sh -O miniconda.sh
```

Comment:
@stutiibm  Why we need to use conda environment, I dont see we installing any conda package

Reply:
Replaced with python virtual environment.